.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+Please note that pull requests without a rationale and clear improvement are
+closed immediately.
+
+Bitcoin Core has a thorough review process and even the most trivial change
```

Comment:
I'd move this to the end. The next paragraph 'Please provide clear motivation...' is more important.